dist/index.custom-properties.css

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Not sure why .css files don't get formatted correctly. Any idea?

Reply:
css files aren't included in sewing-kit's glob when calling stylelint https://github.com/Shopify/sewing-kit/blob/master/src/tools/stylelint/index.ts#L38